packages/horizon/contracts/interfaces/ITAPCollector.sol

```diff
@@ -54,6 +93,50 @@ interface ITAPCollector is IPaymentsCollector {
         bytes signature
     );
 
+    /**
+     * Thrown when the signer is already authorized
+     * @param signer The address of the signer
+     * @param authorizingPayer The address of the payer authorizing the signer
+     */
+    error TAPCollectorSignerAlreadyAuthorized(address signer, address authorizingPayer);
```

Comment:
nit: same as with events, it'd be good to standardize on the ordering: payer, signer